pkg/lso/localvolumediscovery.go

```diff
@@ -0,0 +1,227 @@
+package lso
```

Comment:
I'm not familiar with lso package.
Can we Create localVolumeDiscovery object manually using >oc create ?

Reply:
yes

Reply:
Can you please add Create function?

Reply:
it was already added